src/util.h

```diff
@@ -171,7 +172,7 @@ bool TryCreateDirectories(const fs::path& p);
 fs::path GetDefaultDataDir();
 const fs::path &GetDataDir(bool fNetSpecific = true);
 void ClearDatadirCache();
-fs::path GetConfigFile(const std::string& confPath);
+fs::path GetConfigFile(const std::string& confPath, bool fNetSpecific);
```

Comment:
Set default value `bool fNetSpecific = false`? In that case remove changes where it's false.

Reply:
The "Be explicit whether config file is net specific" commit removes the fNetSpecific=false default argument, because otherwise it would have the opposite default to GetDataDir() which seems confusing to me.

Reply:
Err, the implication being that just dropping/not merging that commit is how you'd get the fNetSpecific=false default.